.github/workflows/run_chat_analyze.yaml

```
@@ -0,0 +1,42 @@
+    - name: Create .env file
+      run: |
+        echo "API_KEY=${{ secrets.GOOGLE_API_KEY }}" >> .env
+
+    - name: Run script
+      run: |
+        python main.py "https://www.youtube.com/watch?v=ff5BbAFKPnI"
+
+    - name: Upload generated files
+      uses: actions/upload-artifact@v3
+      with:
+        name: generated-images
```

yamllint (1.29.0-1)

[warning] wrong indentation: expected 10 but found 8

(indentation)